### PR TITLE
LPD-15578 Make login fixture support different types of users

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:playwright

**Comment by izaera** [comment]
ci:stop

**Comment by izaera** [comment]
Setting to draft as I'm going to change the local setup script.

**Comment by izaera** [comment]
ci:test:playwright

File: modules/test/playwright/tools/test-setup.sh [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+function copy_portal_ext_properties() {
+	for file in \
+		"${PROJECT_DIR}/modules/test/playwright/tests/${TEST_PROJECT}/env/portal-ext.properties" \
+		"${PROJECT_DIR}/modules/test/playwright/env/portal-ext.properties"
+	do
+		if [ -f "${file}" ]
+		then
+			echo "  ===> ${file}"
+
+			cp "${file}" "${BUNDLES_DIR}"
```
Comment (dsanz) [inline modules/test/playwright/tools/test-setup.sh]: would'nt this overwrite one file with the other? same name, same target directory. Is this intended?

**Comment by izaera** [comment]
ci:test:sf

File: modules/test/playwright/tools/test-setup.sh (hunk 1 above)
Comment (izaera) [reply modules/test/playwright/tools/test-setup.sh]: Outdated: I have removed the bash scripts.

**Comment by izaera** [comment]
Amazing: SF succeeds locally, but complains about typescript compilation in CI :flushed:

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:playwright

**Comment by izaera** [comment]
ci:test:playwright

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:playwright

**Comment by izaera** [comment]
I've assesed that the failures in Playwright are happening in master too, so this should be green as far as playwright tests are concerned...

**Comment by izaera** [comment]
ci:forward

**Comment by izaera** [comment]
ci:test:stable

**Comment by izaera** [comment]
The unique failures between two successive runs are different and seem unrelated so I deem this PR safe.

I'll rerun stable again because I don't want to forward this until it is properly documented to avoid interfering with other people's work. So while I write the docs we can run stable once more to see if it gets green.

**Comment by izaera** [comment]
ci:test:stable

**Comment by izaera** [comment]
ci:forward:force

**Comment by izaera** [comment]
ci:test:playwright

**Comment by izaera** [comment]
ci:test:relevant

**Comment by izaera** [comment]
ci:forward:force

**Pull request closed without merging**